magda-web-client/src/UI/Notification.scss

```diff
@@ -1,22 +1,30 @@
 @import "../variables";
-.notification {
+
+.notification-box {
     position: fixed;
     height: 100vh;
     width: 100vw;
     left: 0;
     top: 0;
     background: rgba(0, 0, 0, 0.5);
-    z-index: 999;
-}
+    z-index: 9999;
+
+    .notification__inner {
+        position: fixed;
+        right: 14px;
+        bottom: 24px;
+        padding: 16px;
+        max-width: 400px;
+        min-width: 290px;
+    }
+
+    h3 {
+        word-break: keep-all;
+        word-wrap: normal;
+        margin-right: 32px;
+        text-align: left;
```

Comment:
It would be nicer to remove the margins on h3, something like `margin: 0 32px 0 0;`
<img width="320" alt="screen shot 2018-05-14 at 4 37 12 pm" src="https://user-images.githubusercontent.com/7508939/39981719-17042782-5795-11e8-9b89-25bee46b326c.png">